src/v2/Apps/ViewingRoom/Components/ViewingRoomsFeaturedRail.tsx

```diff
@@ -1,7 +1,9 @@
 import React from "react"
-import { Box, Sans } from "@artsy/palette"
+import { Box, Flex, Link, MediumCard, Sans, Spacer } from "@artsy/palette"
 import { ViewingRoomsFeaturedRail_featuredViewingRooms } from "v2/__generated__/ViewingRoomsFeaturedRail_featuredViewingRooms.graphql"
 import { createFragmentContainer, graphql } from "react-relay"
+import { ViewingRoomCarousel } from "../Routes/Works/Components/ViewingRoomCarousel"
```

Comment:
This could probably be moved to `Apps/ViewingRoom/Components` now that it's shared in a few places